ors-engine/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperConfig.java

```diff
@@ -26,6 +26,7 @@ public class ORSGraphHopperConfig extends GraphHopperConfig {
     private List<CHProfile> coreProfiles = new ArrayList<>();
     private List<LMProfile> coreLMProfiles = new ArrayList<>();
     private List<Profile> fastisochroneProfiles = new ArrayList<>();
+    private static final org.apache.log4j.Logger LOGGER = org.apache.log4j.Logger.getLogger(ORSGraphHopperConfig.class.getName());
```

Comment:
This is a fully qualified import. Better to import the library directly at the top with the others.